src/kernel/include/shared/cos_types.h

```diff
@@ -255,7 +255,7 @@ enum
                                                          CAPMAX_ENTRY_SZ),
 	BOOT_CAPTBL_SELF_INITHW_BASE   = round_up_to_pow2(BOOT_CAPTBL_SELF_INITRCV_BASE + NUM_CPU_COS * CAP64B_IDSZ,
                                                         CAPMAX_ENTRY_SZ),
-	BOOT_CAPTBL_SINV_CAP           = BOOT_CAPTBL_SELF_INITHW_BASE + CAP32B_IDSZ,
+	BOOT_CAPTBL_SINV_CAP           = BOOT_CAPTBL_SELF_INITHW_BASE + CAP64B_IDSZ,
```

Comment:
You're changing the offset of `SINV_CAP` and not the offset of `LAST_CAP` which you want to be based on the new size of `SINV_CAP`.

Reply:
oh wow, great point. I'll fix now.